myriad-scheduler/src/test/java/com/ebay/myriad/scheduler/SchedulerUtilsSpec.groovy

```diff
@@ -29,6 +30,30 @@ class SchedulerUtilsSpec extends Specification {
 
     }
 
+    def "is eligible for FGS"() {
```

Comment:
FGS is too terse IMO.  it makes sense now but 1 year from now no one will know what this is testing.